api/MutationEvent.json

```
@@ -0,0 +1,341 @@
+{
+  "api": {
+    "MutationEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationEvent",
+        "spec_url": "https://w3c.github.io/uievents/#interface-mutationevent",
+        "support": {
+          "chrome": {
+            "version_added": "1"
+          },
+          "chrome_android": {
+            "version_added": "18"
+          },
+          "edge": {
+            "version_added": "12"
+          },
+          "firefox": {
+            "version_added": "1"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": "≤12.1"
+          },
+          "opera_android": {
+            "version_added": "≤12.1"
+          },
+          "safari": {
+            "version_added": "≤4"
+          },
+          "safari_ios": {
+            "version_added": "≤3"
+          },
+          "samsunginternet_android": {
+            "version_added": "1.0"
+          },
+          "webview_android": {
+            "version_added": "≤37"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      },
+      "attrChange": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/uievents/#dom-mutationevent-attrchange",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "≤15"
+            },
+            "opera_android": {
+              "version_added": "≤14"
+            },
```

I sent https://github.com/foolip/mdn-bcd-collector/pull/1297 to make it possible to test the attributes in Presto, and with https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/MutationEvent I can confirm all the members are in Opera 12.16.

```suggestion
            "opera": {
              "version_added": "≤12.1"
            },
            "opera_android": {
              "version_added": "≤12.1"
            },
```